core/trino-main/src/main/java/io/trino/operator/DirectExchangeClientStatus.java

```diff
@@ -147,7 +159,8 @@ public DirectExchangeClientStatus mergeWith(DirectExchangeClientStatus other)
                 spilledPages + other.spilledPages,
                 spilledBytes + other.spilledBytes,
                 noMoreLocations && other.noMoreLocations, // if at least one has some locations, mergee has some too
-                ImmutableList.of()); // pageBufferClientStatuses may be long, so we don't want to combine the lists
+                ImmutableList.of(), // pageBufferClientStatuses may be long, so we don't want to combine the lists
+                getClientWaitingTimeInMillis().mergeWith(other.getClientWaitingTimeInMillis()));
```

Comment:
add a comment why this is correct similar as for `(bufferedBytes + other.bufferedBytes) / 2`

Reply:
I can do it, but for sure? Merging two `TDigest`'s for the same observation / stats is correct by its nature, so I would avoid adding a such comment? Like there is no comment for `spilledBytes + other.spilledBytes`